src/preact/base-element.js

```diff
@@ -81,6 +81,13 @@ export class PreactBaseElement extends AMP.BaseElement {
 
     /** @private {boolean} */
     this.mounted_ = true;
+
+    /** @private {MutationObserver|null} */
+    this.observer_ = this.constructor['passthroughNonEmpty']
+      ? new MutationObserver(
+          this.constructor['passthroughNonEmpty']['callback'].bind(this)
```

Comment:
The callback also will never change. It will always be `() => this.scheduleRender_();`. If it does changes at some point in the future - we'll adjust as needed. But to start with, we want as re-enterable as possible approach here.

Reply:
Sounds good, updated!